cfgov/jinja2/v1/_includes/organisms/footer.html

```diff
@@ -21,10 +21,10 @@
 
             <a class="btn
                       btn__secondary
-                      js-return-to-top
                       o-footer_top-button"
-               href="#">
-                Back to top <span class="cf-icon cf-icon-arrow-up"></span>
+                data-js-hook="behavior_return-to-top"
+                href="#">
+                Back to top <span clasbs="cf-icon cf-icon-arrow-up"></span>
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/pull/2115/commits/6d1c6acdb567feaecb0248214349bd32cf794dd2